### DISCUSSION TITLE
Transfer ownership

### DISCUSSION THREAD

**Discussion by gbrail** [mozilla/rhino#1424] [General]
Hello -- in another discussion, we have been discussing whether it makes sense to transfer this repo from the "Mozilla" GitHub organization to a stand-alone GitHub org.

Mozilla is willing to do this, but generally they don't just transfer IP from Mozilla into the ether -- they prefer that we transfer the ownership of the Rhino IP to some legal entity. I agree -- I don't want to be the maintainer of something that doesn't have clear protections against all of the crazy legal things that happen in the world. (I'm happy to elaborate if people would like, but, for example, I wouldn't want to be held personally liable in some country because of a security vulnerability discovered in Rhino.)

It sounds like we have a few choices:

1. Transfer Rhino to the [Linux Foundation](https://www.linuxfoundation.org/projects/hosting). We could get access to some of their resources that way, and they provide legal frameworks and guidance if we ever need it. It sounds pretty easy to apply. We'd be able to have our own GitHub org.
2. Transfer Rhino to the [Open Source Collective](https://opencollective.com/). This would provide similar benefits to the Linux Foundation, and is similarly easy (if not even easier) to join. We'd also be able to have our own GitHub org.
3. Transfer Rhino to the Apache Software Foundation. I actually think that's quite a bit more work, as ASF expects a certain style of governance, etc. I'm not sure that makes sense but I'm including it to be complete.
4. Do nothing and leave Rhino with Mozilla. There's not much downside to this, except for a bit of overhead when I want to do GitHub admin stuff, but Mozilla isn't really doing anything to help the project any more except for providing us with GitHub and theoretically some legal cover.

I'm curious how other people in the community feel about this.

**Comment by Jaxo** [comment]
Linux Foundation.

**Comment by StSchnell** [comment]
Hello @gbrail,
thank you very much for your activities. Clarifying the ownership is important to create a stable base from this perspective. Unfortunately, I don't know enough of the details to be able to help here, so I can only provide "moral support" in this case. Thanks again for your activities.
Best regards
Stefan

**Comment by michbarsinai** [comment]
Does the Linux Foundation have any limitation on the license of the software being developed? I did not find any limitations on their site, but I might have missed some fine print. 

I know that the Play Framework project transferred itself to Open Source Collective a couple of years ago, and they generally have a good experience.

I think being under Mozilla's umbrella does have a branding advantage for the project, as it's conceived as a positive organization with a considerable clout in the JavaScript community. But I'm not sure if it's worth the admin overhead, or if others even share my view on this :-)

(also - thanks @gbrail for your work on this!)

**Comment by Jaxo** [comment]
May 2024.  1.7.15 is on stage!  Thanks Greg.

**Comment by p-bakker** [comment]
I was under the impression that we could stay under the Mozilla' umbrella, while hosting our repo in its own GitHub org.

Some of the questions [here](https://wiki.mozilla.org/GitHub#Recommendations_and_FAQ) seem to suggest it's an option at least
